docs/intro.md

```diff
@@ -107,8 +107,8 @@ The following table lists four key workflows in the UID2 framework with links to
 | [Workflow for Advertisers](overviews/overview-advertisers.md#workflow-for-advertisers) and [Workflow for Data Providers](overviews/overview-data-providers.md#workflow-for-data-providers) | Organizations that collect user data and push it to DSPs.                                                                                                                                                                                                                                                                                                           | [Advertiser/Data Provider Integration Guide](guides/advertiser-dataprovider-guide.md)                                                                                                                                                          |
-| [Workflow for Publishers](overviews/overview-publishers.md#workflow-for-publishers) | Organizations that propagate UID2 tokens to the bid stream via SSPs.<br/> NOTE: Publishers can choose to leverage the UID2 SDK for JavaScript (see [JavaScript Express Integration Guide](publisher-client-side.md) or [UID2 SDK for JavaScript Reference Guide](sdks/client-side-identity.md)) or complete their own server-only integration without using an SDK. | [JavaScript Express Integration Guide](publisher-client-side.md)<br/>[JavaScript Standard Integration Guide](guides/integration-javascript-standard.md)<br/>[Publisher Integration Guide, Server-Only](guides/custom-publisher-integration.md) |
-| [Opt-Out Workflow](getting-started/gs-opt-out.md#opt-out-workflow) | Consumers who engage with publishers or their SSO providers and other identity providers.                                                                                                                                                                                                                                                                           | N/A                                                                                                                                                                                                                                            |
+| [Workflow for Publishers](overviews/overview-publishers.md#workflow-for-publishers) | Organizations that propagate UID2 tokens to the bid stream via SSPs.<br/> NOTE: Publishers can choose to integrate using Prebid, leverage the UID2 SDK for JavaScript, or complete their own server-only integration without using an SDK. | [Prebid.js Express Integration Guide](guides/integration-prebid.md)<br/>[JavaScript Express Integration Guide](guides/publisher-client-side.md)<br/>[JavaScript Standard Integration Guide](guides/integration-javascript-standard.md)<br/>[Publisher Integration Guide, Server-Only](guides/custom-publisher-integration.md) |
```

Comment:
@genwhittTTD the Prebid.js advanced guide is missing here?

Reply:
@sunnywu good catch thx. But that doc list is so dense. Instead in that column I replaces with a link to the applicable section in the integration guides summary page. Simpler. I did it even for DSP which only has one guide... for extensibility, we will have more!